### PR TITLE
Added a fix to the requirements-dev.txt so that GitHub Actions will work with Black update

### PR DESCRIPTION
**Pull request by jkgoodrich**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -60,23 +59,21 @@ pytest==6.2.5
     # via -r requirements-dev.in
 pyyaml==6.0
     # via pre-commit
-regex==2021.11.10
```
Comment (klaricch) [inline requirements-dev.txt:63]: why does this (and typed-ast/appdirs) need to be removed?

**Review by klaricch — APPROVED** [review]

File: requirements-dev.txt (hunk 1 above)
Comment (jkgoodrich) [reply requirements-dev.txt:63]: I'm not actually sure, but assuming that some other requirement needed it, but it's no longer needed, the `requirements-dev.txt` is automatically generated from the `requirements-dev.in` using `pip-compile`

Comment (jkgoodrich) [reply requirements-dev.txt:63]: @nawatts if you have time, can you confirm this is the reason?

Comment (nawatts) [reply requirements-dev.txt:63]: > assuming that some other requirement needed it, but it's no longer needed

Yes, this is the reason. `pip-compile` adds `# via <package>` comments to this file to indicate where transitive dependencies originate from. `regex` and `typed-ast` both had `# via black` comments, so it looks like the newer version doesn't require them anymore.

Comment (jkgoodrich) [reply requirements-dev.txt:63]: Thank you @nawatts!

**Pull request merged**